_data/projects/cordovue.yml

```diff
@@ -0,0 +1,15 @@
+name: Cordovue
+desc: A sample Apache Cordova application using VueJS.
+site: https://github.com/TheMushrr00m/cordovue
+tags:
+- html
+- css
+- javascript
+- js
+- ecmascript
+- vuejs
+- cordova
+- cross-platform
+upforgrabs:
+  name: starter bug
+  link: https://github.com/adobe/brackets/labels/help%20wanted
```

Comment:
Should this be `https://github.com/TheMushrr00m/cordovue/labels/good%20first%20issue/`?

Reply:
Yeah,it's correct, my mistake!